src/ext/pupil-size/code/dataModels/ValidSamplesModel.m

```diff
@@ -144,6 +144,11 @@ function interpAndSmooth(obj)
             valz          = [diff(valid_t_ms);NaN];

```

Comment:
![image](https://user-images.githubusercontent.com/10925826/165339903-dc58c8cf-e359-4ae8-84de-b5f4ce80812c.png)
careful with the character encoding

Reply:
Yes, this is the conflict of LF (Unix) and CRLF (Windows)... This should be resolved in the pull request 130, which converted all the functions into LF and UTF-8.